curacmd/install

```diff
@@ -1 +1,150 @@
-# Curacmd Bootstrap / Install Script
+#!/bin/bash
```

Comment:
I'm not opposed to it in the least bit whatsoever.  I have honestly somehow never written any bash scripts in my entire career (no idea how that happened).  I just assumed all the cool kids were dropping extensions from their `curl http://interwebz/install` script and thought it was some new trend I was missing out on.  I'm pro `.sh` extension actually but thought it might show my age.